docs/store/datastore.md

```diff
@@ -310,11 +319,14 @@ ParquetTarget(path="ds://test_profile/aws_bucket/path/to/parquet.pq")
 - `name` &mdash; Name of the profile
 - `v3io_access_key` &mdash; Optional. Access key to the remote Iguazio cluster. If not provided, the default is value is taken from the environment variable "V3IO_ACCESS_KEY". For privacy reasons, it's tagged as a private attribute.
 
-## See also
-- {py:class}`~mlrun.projects.MlrunProject.list_datastore_profiles` 
-- {py:class}`~mlrun.projects.MlrunProject.get_datastore_profile`
-- {py:class}`~mlrun.datastore.datastore_profile.register_temporary_client_datastore_profile`
-- {py:class}`~mlrun.projects.MlrunProject.delete_datastore_profile`
-
-The methods `get_datastore_profile()` and `list_datastore_profiles()` only return public information about 
-the profiles. Access to private attributes is restricted to applications running in Kubernetes pods.
+## Adding a data store profile
```

Comment:
Checking the formal docs there is not info for developers.
So keeping with this, I think "adding data store profile" (and adding datastore) should go to a separate directory (docs/internal/datastores ?), and being pointed from CONTRIBUTING.md

Reply:
Developers' docs stay in the build since there is no other hone for them

Reply:
well, i'm not in agreement with that - it's the same as CONTRIBUTING.md which is not mentioned in the formal docs.
Information for developers shouldn't be mixed into the user documentation - it's for different audience with different technical knowledge and in my opinion it ruins the quality of the docs